tests-node/unit/broccoli/translation-reducer/lint-test.js

```diff
@@ -13,18 +13,29 @@ describe('linting', function () {
 
     this.icuFixture = {
       de: {
-        foo: 'Hallo {who}',
+        greeting: 'Hallo {who}',
         sameArgumentDifferentContent: 'You have {numPhotos, plural, =0 {no photos.} =1 {one photo.} other {# photos.}}',
         missingArg: 'You have {numPhotos, plural, =0 {foo} =1 {bar} other {# baz}}',
-        deep: { nested: { ok: 'ok {reason}' } },
+        select: '{gender, select, male {He avoids bugs} female {She avoids bugs} other {They avoid bugs}}',
+        deep: {
+          nested: {
+            ok: 'Account created {reason}',
+            missing: 'Account invalid {reason}, {code}',
+          },
+        },
       },
       en: {
-        foo: 'Hello {whos}',
+        greeting: 'Hello {whos}',
         sameArgumentDifferentContent: 'You have {numPhotos, plural, =0 {foo} =1 {bar} other {# baz}}',
         missingArg: 'You have photos',
-        date: '{datetime, date, long}',
```

Comment:
Curious why the date/time keys were removed from the assertion

Reply:
They seemed redundant. There were already other keys present/missing between the two languages. But happy to bring them back if you'd like.

Reply:
It was indirectly testing https://github.com/ember-intl/ember-intl/blob/master/lib/broccoli/translation-reducer/linter/extract-icu-arguments.js#L11-L12 but that's fine - keep it removed, I'll add a unit test for that

Reply:
👍